src/components/separator/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { FC } from 'react';
+import {
+  Separator as ChakraSeparator,
+  type SeparatorProps,
+} from '@chakra-ui/react';
+
+export const Separator: FC<SeparatorProps> = (props: SeparatorProps) => (
+  <ChakraSeparator {...props} />
+);
+
+export type { SeparatorProps };
```

Comment:
Since this is plain re-export there is no need to create wrappers. You can do:
```suggestion
export { Separator, type SeparatorProps } from '@chakra-ui/react';
```

Reply:
@Averethel when I re-export the component it shows `<React.ForwardRef orientation="horizontal" />` instead of `<Separator orientation="horizontal" />` in Storybook preview. 

Is there a workaround for this?

Reply:
Yes, you can set a `displayName` property on the component. Check [`Grid`](https://github.com/smg-automotive/components-pkg/blob/chakra-v3/root/src/components/grid/index.tsx) for an example